### ISSUE TITLE
Resnet 152  - Loss is not reducing 

### ISSUE THREAD

**Issue by suresh-s** [liuch37/pan-pytorch#9]
I have used backbone ResNet 152 for training with neck_channel = (256, 512, 1024, 2048) with single GPU Tesla T4.  I have used pretrained model taken from here"https://download.pytorch.org/models/resnet152-394f9c45.pth"

Loss is not reducing as epoch increases.   Could you tell me if anything i missed?

training started with this command "**python train.py --batch 6 --epoch 10000 --dataset_type tt --gpu True**"

here below i am giving last 3 epochs logs 

```
Epoch: [313 | 10000]
(1/209) LR: 0.000972 | Loss: 0.110 | Loss (text/kernel/emb): 0.063/0.037/0.010 | IoU (text/kernel): 0.905/0.802
(21/209) LR: 0.000972 | Loss: 0.127 | Loss (text/kernel/emb): 0.071/0.042/0.015 | IoU (text/kernel): 0.906/0.780
(41/209) LR: 0.000972 | Loss: 0.141 | Loss (text/kernel/emb): 0.081/0.046/0.014 | IoU (text/kernel): 0.901/0.772
(61/209) LR: 0.000972 | Loss: 0.137 | Loss (text/kernel/emb): 0.078/0.045/0.014 | IoU (text/kernel): 0.903/0.772
(81/209) LR: 0.000972 | Loss: 0.136 | Loss (text/kernel/emb): 0.078/0.045/0.013 | IoU (text/kernel): 0.903/0.773
(101/209) LR: 0.000972 | Loss: 0.133 | Loss (text/kernel/emb): 0.076/0.045/0.013 | IoU (text/kernel): 0.904/0.775
(121/209) LR: 0.000972 | Loss: 0.133 | Loss (text/kernel/emb): 0.076/0.045/0.013 | IoU (text/kernel): 0.904/0.774
(141/209) LR: 0.000972 | Loss: 0.134 | Loss (text/kernel/emb): 0.077/0.045/0.013 | IoU (text/kernel): 0.903/0.774
(161/209) LR: 0.000972 | Loss: 0.133 | Loss (text/kernel/emb): 0.076/0.045/0.012 | IoU (text/kernel): 0.904/0.774
(181/209) LR: 0.000972 | Loss: 0.131 | Loss (text/kernel/emb): 0.075/0.045/0.012 | IoU (text/kernel): 0.905/0.775
(201/209) LR: 0.000972 | Loss: 0.130 | Loss (text/kernel/emb): 0.074/0.044/0.012 | IoU (text/kernel): 0.905/0.776
Epoch: [314 | 10000]
(1/209) LR: 0.000972 | Loss: 0.104 | Loss (text/kernel/emb): 0.059/0.040/0.005 | IoU (text/kernel): 0.916/0.781
(21/209) LR: 0.000972 | Loss: 0.131 | Loss (text/kernel/emb): 0.077/0.044/0.009 | IoU (text/kernel): 0.903/0.769
(41/209) LR: 0.000972 | Loss: 0.129 | Loss (text/kernel/emb): 0.075/0.045/0.009 | IoU (text/kernel): 0.906/0.774
(61/209) LR: 0.000972 | Loss: 0.129 | Loss (text/kernel/emb): 0.074/0.045/0.010 | IoU (text/kernel): 0.907/0.775
(81/209) LR: 0.000972 | Loss: 0.126 | Loss (text/kernel/emb): 0.071/0.044/0.010 | IoU (text/kernel): 0.908/0.777
(101/209) LR: 0.000972 | Loss: 0.128 | Loss (text/kernel/emb): 0.073/0.045/0.010 | IoU (text/kernel): 0.908/0.775
(121/209) LR: 0.000972 | Loss: 0.126 | Loss (text/kernel/emb): 0.071/0.044/0.010 | IoU (text/kernel): 0.908/0.777
(141/209) LR: 0.000972 | Loss: 0.127 | Loss (text/kernel/emb): 0.072/0.045/0.010 | IoU (text/kernel): 0.908/0.776
(161/209) LR: 0.000972 | Loss: 0.127 | Loss (text/kernel/emb): 0.072/0.044/0.010 | IoU (text/kernel): 0.907/0.776
(181/209) LR: 0.000972 | Loss: 0.129 | Loss (text/kernel/emb): 0.074/0.045/0.010 | IoU (text/kernel): 0.907/0.775
(201/209) LR: 0.000972 | Loss: 0.129 | Loss (text/kernel/emb): 0.074/0.045/0.010 | IoU (text/kernel): 0.906/0.775
Epoch: [315 | 10000]
(1/209) LR: 0.000972 | Loss: 0.102 | Loss (text/kernel/emb): 0.053/0.039/0.011 | IoU (text/kernel): 0.920/0.808
(21/209) LR: 0.000972 | Loss: 0.157 | Loss (text/kernel/emb): 0.092/0.052/0.013 | IoU (text/kernel): 0.900/0.771
(41/209) LR: 0.000972 | Loss: 0.147 | Loss (text/kernel/emb): 0.084/0.049/0.014 | IoU (text/kernel): 0.901/0.773
(61/209) LR: 0.000972 | Loss: 0.137 | Loss (text/kernel/emb): 0.078/0.046/0.013 | IoU (text/kernel): 0.905/0.779
(81/209) LR: 0.000972 | Loss: 0.136 | Loss (text/kernel/emb): 0.077/0.046/0.013 | IoU (text/kernel): 0.905/0.776
(101/209) LR: 0.000972 | Loss: 0.134 | Loss (text/kernel/emb): 0.076/0.045/0.013 | IoU (text/kernel): 0.906/0.776
(121/209) LR: 0.000972 | Loss: 0.134 | Loss (text/kernel/emb): 0.076/0.046/0.012 | IoU (text/kernel): 0.905/0.775
(141/209) LR: 0.000972 | Loss: 0.134 | Loss (text/kernel/emb): 0.076/0.046/0.012 | IoU (text/kernel): 0.905/0.775
(161/209) LR: 0.000972 | Loss: 0.133 | Loss (text/kernel/emb): 0.076/0.045/0.012 | IoU (text/kernel): 0.905/0.775
(181/209) LR: 0.000972 | Loss: 0.134 | Loss (text/kernel/emb): 0.076/0.046/0.012 | IoU (text/kernel): 0.905/0.775
(201/209) LR: 0.000972 | Loss: 0.133 | Loss (text/kernel/emb): 0.076/0.045/0.012 | IoU (text/kernel): 0.905/0.775
```

**Comment by liuch37** [comment]
The training already converged. You can see IoU reaches 90%. You can evaluate your model and use it now.
